src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function (num) {
+      this.result += num;
+
+      return this;
+    },
+    subtract: function (num) {
+      this.result -= num;
+
+      return this;
+    },
+    multiply: function (num) {
+      this.result *= num;
+
+      return this;
+    },
+    divide: function (num) {
+      this.result /= num;
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - The method shorthand should be used when defining methods in an object. This makes the code more concise and easier to understand.

Instead of:
divide: function(num) {
  // your code
},

Use:
divide(num) {
  // your code
},